tools/fonts/@/gootville/LICENSE.txt

```diff
@@ -0,0 +1,5 @@
+Designed by Grzegorz Pruchniakowski.
```

Comment:
Which version of the files do we have here?

Is it taken from here: https://github.com/musescore/MuseScore/tree/master/fonts/gootville

That readme specifies version 1.2. If this is the version we are using, can we rename the files to Gootville_1.2.otf and GootvilleText_1.2.otf, etc?

This makes the naming consistent with our Bravura and Petaluma font files, and easily allows for future updates to Gootville.

Reply:
versions updated